### ISSUE TITLE
Ensure tables display well on mobile

### ISSUE THREAD

**Issue by crwallace** [department-of-veterans-affairs/vets-website#2693]
Text wraps cleanly and has adequate leading and padding to be legible

**Comment by melwoodard** [comment]
#2692 will solve the leading and padding issues, but sizing tables down to mobile width is still an issue - @bshyong I'd like your thoughts on methods to solve for this - the toughest table we're dealing with is the VEC Employer Commitments search page.

**Comment by bshyong** [comment]
@melwoodard tables are always pretty difficult... There's not much we can do on mobile for very dense information (lots of columns)—a higher level question is whether there is a better way to present the info on mobile instead of using a dense table.

That said, two ways to help improve the tables are A. horizontal scrolling and B. making the table display different on mobile (much more involved implementation)

See the following screenshot:
![screen shot 2016-08-23 at 10 19 02 am](https://cloud.githubusercontent.com/assets/303289/17902296/93f4c5ea-691b-11e6-9233-f969933619b8.png)

**Comment by melwoodard** [comment]
@bshyong agreed - I chatted with Molly yesterday (who is currently exploring brand refinement for vets.gov), and we came to the same conclusion - it's 2016 and we really should be looking for better ways to represent this type information. 

I agree with your short-term solutions - in my opinion we should add horizontal scrolling for now to make this more usable.

In the long term we can collaborate with the content team to explore better UI patterns for dense data display / comparisons.
